### DISCUSSION TITLE
Potion Fluids feature

### DISCUSSION THREAD

**Discussion by Buuz135** [neoforged/NeoForge#1537] [Ideas]
As same as milk, there are several mods that add potion fluids for crafting and it would be pretty easy to add a fluid that used the Potion component.

As there is always discussion about numbers, let's just use full buckets (1 bucket = 1 potion item), filling bottles from water sources is done for free and water is infinte, noone is struggling to get water in the game.

There doesn't need to be a way to transfer bottle to fluid.

Before anyone says anything about cauldrons, they suck, if you use it fill water bottles this is not how you get rich in 2027.

**Comment by TelepathicGrunt** [comment]
Personally, I’m in the opposite camp of lets kill off Milk fluid in 1.22 lol

**Comment by TelepathicGrunt** [comment]
A bucket equaling a single bottle is confusing and such a stark contrast in container sizes visually. At the very least, make it so 1 bucket equals 4 bottles
